test/unit/api/solver_black.h

```diff
@@ -141,6 +143,16 @@ class SolverBlack : public CxxTest::TestSuite
   void testPrintModel2();
   void testPrintModel3();
 
+  void testBlockModel1();
+  void testBlockModel2();
+  void testBlockModel3();
+  void testBlockModel4();
+  void testBlockModelValues1();
+  void testBlockModelValues2();
+  void testBlockModelValues3();
+  void testBlockModelValues4();
+  void testBlockModelValues5();
+
```

Comment:
Test for `expandDefinitions` is missing.

Reply:
Not applicable anymore.